tests/Api/Shop/PaymentRequestsTest.php

```diff
@@ -79,7 +79,6 @@ public function it_creates_a_payment_request(array $fixturesPaths, string $respo
             content: json_encode([
                 'paymentId' => $payment->getId(),
                 'paymentMethodCode' => $payment->getMethod()->getCode(),
-                'action' => 'capture',
```

Comment:
Why, after removing this, is there still a `capture` action in the response? 🤔

Reply:
The default action will be set by `sylius.provider.payment_request.default_action` afterward.
By default, it will be a `capture` but if the gateway config is using `use_authorize` set to `true`, then it will be set to `authorize`.
If the payment method is not supporting `capture` or `authorize` then it can choose something else like `subscribe` or `something_else` as the default action.

Reply:
@GSadee: Now I'm reviewing this, it would be nice if when a null action is provided to this validator, we should get the default action and continue trying to check if a command provider exists for it. This way no not processable payment request will be created in the database.